src/shell/shell.cpp

```diff
@@ -202,6 +202,11 @@ uint16_t get_tick_random_number() {
 	return (uint16_t)(GetTicks() % random_uplimit);
 }
 
+// Yo dawg MSVC-Clang likes to complain about pragmas
+// So here's a pragma so you stop complaining about pragmas
```

Comment:
Funny 😄 But if you do the per-line suppression thing by the magic PVS-Studio specific comment, you won't need this 😄